src/cctz_benchmark.cc

```diff
@@ -101,6 +101,7 @@ const char RFC3339_sec[] = "%Y-%m-%d%ET%H:%M:%S%Ez";
 const char RFC1123_full[] = "%a, %d %b %Y %H:%M:%S %z";
 const char RFC1123_no_wday[] = "%d %b %Y %H:%M:%S %z";
 
+// clang-format off
```

Comment:
It seems preferable to put this inside the list, like the example in `src/civil_time_test.cc`.

Reply:
Done.